app/models/user.rb

```diff
@@ -65,7 +65,7 @@ class User < ActiveRecord::Base
 
   validates_each :username do |record,attr,value|
     if BANNED_USERNAMES.include?(value.to_s.downcase)
-      record.errors.add(attr, "is not permitted")
+      record.errors.add(attr, "non valido")
```

Comment:
```suggestion
      record.errors.add(attr, "non permesso")
```

Reply:
in italiano ha un'accezione leggermente diversa. Siccome viene usato con più di una parola, "valido" mi sembrava più generico.

Reply:
Non valido mi dà l'impressione che il problema siano i dati immessi mentre non permesso è qualcosa che non puoi fare.